_projects/2024/100535241/100535241.Rmd

```diff
@@ -0,0 +1,824 @@
+---
+title: "Winners and losers of tax legislation changes"
+description: |
+  Recreating the ABC news article Graph for MCSS Data Viz 24/25
+categories: "2024"
+author: Brad McKenzie
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+    toc_float: true
+    code_folding: true
+---
+
+# About the graph
```

Comment:
The main title of the post is a level-1 heading. So please your top-level headings should be `##`, then `###` etc. for subsections.